CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.1.4]
```

Comment:
At the end of the changelog file, there are some links so that when you click on the [1.1.4] tag, it will open a tab and you will see all the changes made between latest versions. Please update that too